config/serial_udev_ubuntu1804.rules

```diff
@@ -0,0 +1,4 @@
+# make symlinks serial0 and serial1, pointing to the default device names. Grant read/write permissions for the devices
+# note: not sure if these actually ought to be tty, but this works.
+KERNEL=="ttyS0", SYMLINK+="serial0" GROUP="tty" MODE="0660"
+KERNEL=="ttyAMA0", SYMLINK+="serial1" GROUP="tty" MODE="0660"
```

Comment:
I think this is missing a few things, which can also be in the form of separate instructions. The longer udev rules work, or you can add the user to the tty group manually after.

Reply:
Added the command for putting the user in the tty group. Forgot that before.